phpBB/includes/acp/acp_main.php

```diff
@@ -431,13 +431,23 @@ function main($id, $mode)
 
 		if ($auth->acl_get('a_board'))
 		{
+			/** @var \phpbb\version_helper $version_helper */
 			$version_helper = $phpbb_container->get('version_helper');
 			try
 			{
 				$recheck = $request->variable('versioncheck_force', false);
-				$updates_available = $version_helper->get_suggested_updates($recheck);
+				$updates_available = $version_helper->get_update_on_branch($recheck);
+				$upgrades_available = $version_helper->get_suggested_updates();
```

Comment:
No. That would result in force fetching the latest version info twice. The first call will force fetch it and the second call can use that force fetched data.